elpis/gui/src/components/Model/Settings.js

```diff
@@ -92,6 +92,9 @@ class ModelSettings extends Component {
                                         className="attached"
                                         enableReinitialize
                                         initialValues={{
+                                            // This could be a user api key or an organization api key
+                                            huggingface_api_token: "", 
```

Comment:
Are we going to give them the option to provide their own?

Reply:
Yes, that's the plan. After this PR is accepted I'dd add GUI for user & CLI input.

Reply:
Turns out that Avi has done this, the interface has a field for API key, as well as custom model name.